HousingSearchApi/V1/Boundary/Requests/HousingSearchRequest.cs

```diff
@@ -33,8 +33,8 @@ public abstract class HousingSearchRequest
         [FromQuery(Name = "isDesc")]
         public bool IsDesc { get; set; }
 
-        [FromQuery(Name = "matchType")]
-        public Nest.TextQueryType MatchType { get; set; } = Nest.TextQueryType.MostFields;
+        [FromQuery(Name = "simplequery")]
```

Comment:
as this is a bool, should be keep the same naming convention - i.e. isSimpleQuery?

Reply:
Good spot - changed